### PR TITLE
Integrated the HTTP patch for OSX/iOS

### PR DESCRIPTION
**Pull request by SmilyOrg**
Seems to work for me on OSX 10.9.5 and iOS 7.1.2

### CODE REVIEW

File: loom/common/platform/platformHttpOSX.mm [hunk 1]
```diff
@@ -107,13 +107,13 @@ -(void)connectionDidFinishLoading:(NSURLConnection *)connection
     if(cacheToFile != NULL)
         [receivedData writeToFile:[NSString stringWithUTF8String:cacheToFile] atomically:YES];
 
-    NSString *response = nil;
-    response = [[NSString alloc] initWithData:receivedData encoding:NSUTF8StringEncoding];
-
+    utByteArray* bytes = new utByteArray();
+    bytes->attach(receivedData.mutableBytes, receivedData.length);
```
Comment (SmilyOrg) [inline loom/common/platform/platformHttpOSX.mm]: Does it need to get copied instead here or is just reusing the receivedData ok? It seems to work fine as is, but maybe it just has a low repro rate.

Comment (bengarney) [inline loom/common/platform/platformHttpOSX.mm]: Can we do a soak test to check for memory leaks?

Comment (SmilyOrg) [inline loom/common/platform/platformHttpOSX.mm]: Yes we can!

**Comment by bengarney** [comment]
Is there a card for soak test on this @SmilyOrg ?

**Comment by SmilyOrg** [comment]
It's an undone checkbox in the Implement new HTTP byte stuff for OSX and IOS card, but we can make it a separate one.

**Comment by bengarney** [comment]
That's fine as long as tracked.

**Comment by bengarney** [comment]
Failure: https://gist.github.com/bengarney/956c9074281c62b32b00

**Comment by SmilyOrg** [comment]
Green!

**Pull request merged**